phac-nml/irida-next
# Docs: Flesh out ga4gh wes docs

## What does this PR do and why?

This PR adds more information to the GA4GH WES docs

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally

1. Start docs with the following
```bash
cd docs-site
pnpm build
npm run serve
```
2. In a browser, view the new docs pages
3. Check that the information makes sense from an IRIDA Next admin perspective.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
